src/style.scss

```diff
@@ -121,21 +121,32 @@
   overflow: hidden;
   height: 0;
   opacity: 0;
-  visibility: hidden;
   transition: all 0.5s ease-in-out;
-  transition-property: opacity, height, visibility;
+  transition-property: opacity, height;
```

Comment:
This update is adding in a new jerking of the accordion. 


https://github.com/AlaskaAirlines/auro-accordion/assets/181089/78bfb5ec-8a95-4e42-be86-8a1c79ce1842